package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "install": "^0.13.0"
+  }
+}
```

Comment:
Why did you add an `install` dependency in the outer part of frontend and backend folder?
I think it is by mistake, as I can't see this dependency being used anywhere please remove it.
Both this file and package-lock.json which is outside frontend and backend folder needs to be removed.